sci-mathematics/rstudio/rstudio-1.4.1106.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake qmake-utils
+
+QT_VER=5.12.6
+QT_SLOT=5
+
+DESCRIPTION="IDE for the R language"
+HOMEPAGE="
+	https://rstudio.com
+	https://github.com/rstudio/rstudio/"
+SRC_URI="
+	https://github.com/rstudio/rstudio/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://s3.amazonaws.com/rstudio-dictionaries/core-dictionaries.zip
+"
+
+LICENSE="AGPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+# IUSE="dedicated libressl server"
+
+# ADD?
+# libsqlite3-dev (this is the ubuntu thingy) → dev-db/postgres
+# libpq-dev → dev-db/sqlite
+# yaml-cpp something?
+#
+# check docker/jenkins
+RDEPEND="
+	dev-db/postgresql
+	dev-libs/soci[postgres,sqlite]
+	>=dev-libs/boost-1.69:=
+	>=dev-libs/mathjax-2.7
+	>=dev-lang/R-3.0.1
+	>=dev-qt/qtcore-${QT_VER}:${QT_SLOT}
+	>=dev-qt/qtdbus-${QT_VER}:${QT_SLOT}
+	>=dev-qt/qtdeclarative-${QT_VER}:${QT_SLOT}
+	>=dev-qt/qtnetwork-${QT_VER}:${QT_SLOT}
+	>=dev-qt/qtopengl-${QT_VER}:${QT_SLOT}
+	>=dev-qt/qtpositioning-${QT_VER}:${QT_SLOT}
+	>=dev-qt/qtsensors-${QT_VER}:${QT_SLOT}
+	>=dev-qt/qtsql-${QT_VER}:${QT_SLOT}
+	>=dev-qt/qtsvg-${QT_VER}:${QT_SLOT}
+	>=dev-qt/qtwebchannel-${QT_VER}:${QT_SLOT}
+	>=dev-qt/qtwebengine-${QT_VER}:${QT_SLOT}
+	>=dev-qt/qtxml-${QT_VER}:${QT_SLOT}
+	>=dev-qt/qtxmlpatterns-${QT_VER}:${QT_SLOT}
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	app-arch/unzip
+	dev-vcs/git
+"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-1.4.1106-add-missing-boost-namespace-import.patch"
+	"${FILESDIR}/${PN}-1.4.1106-add-missing-boost-bind-import.patch"
+	"${FILESDIR}/${PN}-1.4.1106-fix-server-install-paths.patch"
+	"${FILESDIR}/${PN}-1.4.1106-remove-hardcoded-internal-mathjax-and-pandoc-deps.patch"
+	"${FILESDIR}/${PN}-1.4.1106-remove-mathjax-internal-dep-check.patch"
+	"${FILESDIR}/${PN}-1.4.1106-soci-cmake-find_library.patch"
+)
+
+src_unpack() {
+	unpack ${P}.tar.gz
+	cd "${S}" || die
+	mkdir -p dependencies/common/dictionaries || die
+	unzip -qd dependencies/common/dictionaries \
```

Comment:
Can we call unpack?

Reply:
I'll check! This is how the original ebuild author did it, so I assumed it was necessary

Reply:
Ok, I posted a new attempt using `unpack` let me know your thoughts.

Reply:
I believe you stated in a different comment that you were happy with the latest approach. Closing this as resolved.